README.md

```diff
@@ -27,14 +28,16 @@ This library depends on
 - [``matplotlib``](https://matplotlib.org/stable/)
 - [``urllib3``](https://urllib3.readthedocs.io/en/stable/)
 - [``urchin``](https://github.com/fishbotics/urchin)
+- [``jaxsim``](https://github.com/ami-iit/jaxsim)
 
 To install you can use the following commands
 
 
 ```
-conda create -n comododev -c conda-forge adam-robotics idyntree bipedal-locomotion-framework=0.19.0 mujoco mujoco-python numpy mujoco-python-viewer matplotlib urllib3 urchin resolve-robotics-uri-py notebook
+conda create -n comododev -c conda-forge adam-robotics bipedal-locomotion-framework=0.19.0 mujoco-python-viewer matplotlib urllib3 urchin notebook jaxsim
+
```

Comment:
In the future it would have sense to unify the way in which the different simulator are installed. For drake it is an if instead for both mujoco and jaxsim they are included in the main instruction, I will deal with it in the future

Reply:
Yep I agree with you! In #23 this is already implemented, there is a default environment related only to comodo e and one different environment building on the default one for the specific simulator to be used. We could provide also instructions on how to do this on conda in a next PR.

Reply:
Great !